R/algorithm.R

```diff
@@ -161,6 +161,24 @@ algorithm <- function() {
       title = "Metformin purchases that aren't potentially for the treatment of PCOS",
       logic = "NOT (koen == 2 AND atc =~ '^A10BA02$' AND ((date - foed_dato) < years(40) OR indication_code %in% c('0000092', '0000276', '0000781')))",
       comments = "Woman is defined as 2 in `koen`."
+    ),
+    is_any_t1d_primary_diagnosis = list(
```

Comment:
```suggestion
    has_any_t1d_primary_diagnosis = list(
```
I know this doesn't really fit the naming, but "is any t1d primary diagnosis" sounds weird

Reply:
I like `has_` better too :tada: